tests/lib/app_list.js

```diff
@@ -2,7 +2,9 @@
     Helper functions related to app lists.
     Prominently used in app_list.js and search.js tests.
 */
-var constants = require('./constants');
+var constants = helpers.localRequire('tests/lib/constants');
+var _ = helpers.localRequire('node_modules/underscore/underscore');
```

Comment:
Just a thought, but if these are needed in a couple of places adding them to the shim could be considered.